code/src/index.css

```diff
@@ -5,9 +5,149 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
 }
 
-code {
+/* code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
+} */
+
```

Comment:
An alternative would be to use a css component for each js-component. It makes it easier to track which styling that belongs to the each component.